package-lock.json

```diff
@@ -6,6 +6,7 @@
   "packages": {
     "": {
       "version": "1.2.6",
+      "hasInstallScript": true,
```

Comment:
Oh I think I need to revert this. But what's the plan for avoiding checking in this change going forward?

Reply:
i'm not familiar with this, but it seems harmless